### DISCUSSION TITLE
gesp-1-bcqm3024/

### DISCUSSION THREAD

**Discussion by giscus** [lihongzheshuai/lihongzheshuai.github.io#353] [General]
# gesp-1-bcqm3024/

题型与BCQM3023一样，只是中间计算逻辑不同，而且计算逻辑似乎还变简单了，不过多了点细节要求而已，作为巩固练习。

https://www.coderli.com/gesp-1-bcqm3024/

**Comment by Niuger** [comment]
2024.9.29打卡
